go.mod

```diff
@@ -162,3 +164,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	nhooyr.io/websocket v1.8.7 // indirect
 )
+
+replace go.viam.com/rdk => github.com/viamrobotics/rdk v0.24.0-rc0.0.20240410143850-e4e67584717a
```

Comment:
Upgrade to this commit of RDK: https://github.com/viamrobotics/rdk/pull/3788

Reply:
Do we want to keep this replace statement or just update the require section?

Reply:
I would be in favor of keeping the replace for now and remove the replace when the passthrough features land in a tagged release of RDK (which should be next tuesday)